html_text/id/lzh/sutta/da/da11.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<meta name="author" content="Seniya Ariyakumara">
+<title></title>
+</head>
+<body>
+<div id="text" lang="id">
+<section class="sutta" id="da11">
+<article>
+<div class="hgroup">
+<p class="division" lang="san" translate="no">Dīrghāgama</p>
+<h1>11. Kotbah Meningkat Satu per Satu</h1>
+</div>
+<p>Demikianlah telah kudengar. Suatu ketika Sang Buddha sedang berdiam di Śrāvastī di Hutan Jeta, taman Anāthapiṇḍada, bersama dengan perkumpulan besar dari seribu dua ratus lima puluh orang bhikkhu.</p>
+<p>Pada waktu itu Sang Bhagavā berkata kepada para bhikkhu: “Aku akan mengajarkan kalian Dhamma yang mulia, yang ungkapannya pada awal, pertengahan, dan akhirnya adalah benar keseluruhannya, yang mengandung makna dan diberkahi dengan kemurnian kehidupan suci, yaitu, hal-hal yang meningkat satu per satu. Dengarkanlah dan perhatikan apa yang akan kuajarkan kepada kalian.” Kemudian para bhikkhu menerima pengajaran itu dan mendengarkan.</p>
+<p>1. Sang Buddha berkata kepada para bhikkhu: “Hal-hal yang meningkat satu per satu <span class="add">adalah sebagai berikut</span>: satu hal yang membawa keberhasilan besar, satu hal yang seharusnya dilatih, satu hal yang seharusnya dipahami, satu hal yang seharusnya dilenyapkan, dan satu hal yang seharusnya direalisasikan.</p>
```

Comment:
These list items would best go in an ordered list element (I'll expend on this by PM).